akka-http-tests/src/test/scala/akka/http/scaladsl/server/DontLeakActorsOnFailingConnectionSpecs.scala

```diff
@@ -59,22 +60,25 @@ abstract class DontLeakActorsOnFailingConnectionSpecs(poolImplementation: String
             handleResponse(resp, id)
         }
 
-        val running = source.via(clientFlow).runWith(sink)
-        countDown.await(10, TimeUnit.SECONDS) should be(true)
-        Await.result(running, 10.seconds)
-
+        EventFilter.error(pattern = """\d+\: FAIL.*""", occurrences = reqsCount).intercept {
+          EventFilter[StreamTcpException](start = "Outgoing request stream error").intercept {
+            val running = source.via(clientFlow).runWith(sink)
+            countDown.await(10, TimeUnit.SECONDS) should be(true)
+            Await.result(running, 10.seconds)
+          }
+        }
       }
     }
   }
 
   private def handleResponse(httpResp: Try[HttpResponse], id: Int): Unit = {
     httpResp match {
       case Success(httpRes) ⇒
-        system.log.info(s"$id: OK: (${httpRes.status.intValue}")
+        system.log.debug(s"$id: OK: (${httpRes.status.intValue}")
```

Comment:
We might even want to swap the log levels here as an OK result would be unexpected while an error result should be the norm.